Wiring/TickSpec.Xunit/TickSpec.Xunit.fsproj

```diff
@@ -1,9 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../../netfx.props" />
+  <Import Project="../../nuget.props" />
```

Comment:
cant these be done as Directory.build.props ?

Reply:
Yes. However, I wonder whether it is not better to be explicit and mark the projects we want to pack.

Reply:
I agree Explict is better than Implicit in general. However, I think D.B.P is a well established convention at this point, as are all `dotnet *` commands.